development/editor/introduction_to_editor_development.rst

```diff
@@ -42,6 +42,17 @@ editor builds to decrease the binary size of export templates. See the
 `modules/ <https://github.com/godotengine/godot/tree/master/modules>`__ folder
 in the Godot source repository.
 
+Some important files in the editor are:
+
+- `editor/editor_node.cpp <https://github.com/godotengine/godot/blob/master/editor/editor_node.cpp>`__: Main editor initialization file.
```

Comment:
Maybe it can refer to it as the "main scene", as it's a concept familiar to Godot users, and pretty much what it is.

Reply:
Done. I can also make a `3.3` version of these changes once you approve them.